src/einsteinpy/coordinates/differential.py

```diff
@@ -8,6 +8,7 @@
     SphericalConversion,
 )
 from einsteinpy.coordinates.utils import v0
+from einsteinpy.utils.exceptions import CoordinateError
```

Comment:
Since you imported the error classes into `utils/__init__.py`, you do not need to invoke the full path. Just `einsteinpy.utils` would suffice.